odoo/16.0/templates/odoo.conf.tmpl

```diff
@@ -32,7 +32,7 @@ logfile = {{ default "None" .Env.ODOO_LOGFILE }}
 log_handler = {{ default ":INFO" .Env.ODOO_LOG_HANDLER }}
 log_level = {{ default "info" .Env.ODOO_LOG_LEVEL }}
 logrotate = {{ default "False" .Env.ODOO_LOGROTATE }}
-longpolling_port = {{ default "8072" .Env.ODOO_LONGPOLLING_PORT }}
+gevent_port = {{ default "8072" .Env.ODOO_LONGPOLLING_PORT }}
```

Comment:
```suggestion
gevent_port = {{ default "8072" .Env.ODOO_GEVENT_PORT }}
```

Reply:
I worried that would be a breaking change.
But if you're OK with it it needs to be changed in other locations too.

Reply:
Done.